src/content/docs/en/tutorial/5-astro-api/2.mdx

```diff
@@ -89,8 +89,7 @@ You can create entire sets of pages dynamically using `.astro` files that export
 
 2. Filter your list of posts to only include posts that contain the page's own tag.
 
-    ```astro title="/src/pages/tags/[tag].astro" ins={4}
-    ---
+    ```astro title="src/pages/tags/[tag].astro" ins={3}
```

Comment:
Thanks @ovidozio, I'll point out we probably do want the frontmatter delimiter since that's part of the Astro syntax and we do keep it in the other code samples
````suggestion
    ```astro title="src/pages/tags/[tag].astro" ins={4}
    ---
````